Merge/The Merge: Consensus API design space.md

```diff
@@ -0,0 +1,523 @@
+
+**Speaker 11**
+* Okay, so yeah, back through these like I do see while you in, if we say that, we propagate all this information to the folk Choice like this informations information to the execution client sends, you will not get back to these pattern for any other data. So that might be valuable and it might be reasonable to do. Once and also this, this unification requires the corresponding, updating the 3675, because there are two like events that this method maps on.
+
+**Mikhail** 
+* Yep. Yeah, we have like four minutes and it was we're going to talk about the scene. But I think we have not, we have not enough time for this. So I propose to reach out the Geth team in this Discord to talk about this in. Go find the requirements of the same process on the, on the API. And what do you think about making a call? What is the better time for? The next goal is, like, one week or Weeks from from from this.
+
+**Speaker 12**
+* So we're currently working on our part of this Spec of what we think about the sync, which guarantees we can provide and Felix is unfortunate not here today, but he is currently writing down a new document with basically everything about about the sync. So I think, in one week, we should be finished. If you you all have, like, time there. I don't know. 
+
+**Mikhail** 
+* Yeah, I would make a call like one week after if I like using the same time slot. So any other opinions on that? 
+
+**Tim**
+* I mean we can use like half of all core devs or two-thirds of all core devs to discuss the same for the merge like we and its basic. It's not exactly one week and it's not exactly the same time but it might be a good enough place. 
+
+**Mikhail** 
+* If we can do this that the awesome sure. Yeah, it's hard to see what's higher priority.
+
+**Tim**
+* Yeah, so I guess yeah just in terms of next steps. Like as soon as they get team has like a sync right up just posted in the El Cortez agenda and we'll make sure to cover that first on the call next week. 
+
+**Mikhail** 
+* Yep, and whatever time will tell, whatever time periods. It's really awkward to have. So we'll have to go through. We'll just try to do as much as we can. Like, I mean just follow this discussion there. Okay, I'm stop sharing. Thanks everyone for coming. I was like, expecting, not reaching the end of the document today. Thank you so much. See you later. Exactly.
+
+
+-------------------------------------------
+## Attendees
+**Mikhail Kalinin**  
```

Comment:
```suggestion
* Mikhail Kalinin  
```

Reply:
Update all attendees in the similar format (Row 354 - 379)

Reply:
Done